/*******************************************************************************
 * Copyright (c) 2014 Elektrobit Automotive GmbH
 * All rights reserved
 *
 * This file is part of project EB RACE
 *
 * Unauthorized copying of this file is strictly prohibited
 *******************************************************************************/
package TargetAgent.Protocol.LinuxAppStats;

option java_package = "com.elektrobit.ebrace.targetdata.adapter.linuxappstats";
option java_outer_classname = "LinuxAppStatsTAProto";


message AppStatistics {
	required string trace 					= 1; 
}

enum Key
{
   APP_STATISTICS_SET_RESOLUTION = 0;
   APP_STATISTICS_TRIGGER 		 = 1;
}

message Entry {
	required Key mKey = 1;
	optional uint32 mValue = 2;
}

message AppConfiguration{
	repeated Entry configEntry = 1;
}